chemprop/v2/cli/convert.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import torch
+import logging
+from pathlib import Path
+from argparse import ArgumentParser, Namespace
+
+from chemprop.v2.cli.utils import Subcommand
+from chemprop.v2.utils.v1_to_v2 import convert_model_v1_to_v2_file
+
+class ConvertSubcommand(Subcommand):
+    COMMAND = "convert"
+    HELP = "convert a v1 model checkpoint (.pt) to a v2 model checkpoint (.ckpt)"
+
+    @classmethod
+    def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
+        return add_convert_args(parser)
+
+    @classmethod
+    def func(cls, args: Namespace):
+        args = process_args(args)
+        main(args)
+
+def add_convert_args(parser: ArgumentParser) -> ArgumentParser:
+    parser.add_argument(
+        "-i",
+        "--input_path",
+        type=str,
+        required=True,
+        help="Path to a v1 model .pt checkpoint file.",
+    )
+    parser.add_argument(
+        "-o",
+        "--output_path",
+        type=str,
+        help="Path at which the converted model will be saved.",
+    )
+    return parser
+
+def process_args(args: Namespace) -> Namespace:
+    args.input_path = Path(args.input_path)
+    args.output_path = Path(args.output_path or Path.cwd() / (args.input_path.stem + "_v2.ckpt"))
+    return args
+
+def main(args: Namespace) -> None:
+    logging.info(f"Converting v1 model checkpoint '{args.input_path}' to v2 model checkpoint '{args.output_path}'...")
```

Comment:
I believe to use logging you need this line after your imports at the top of the script `logger = logging.getLogger(__name__)` and then use `logger.info(...)` instead of `logging.info(...)`.